projects/equity-spot/index.js

```diff
@@ -0,0 +1,32 @@
+// Equity Spot - Central Vaults
+
+const sdk = require("@defillama/sdk")
+const { sumTokens } = require('../helper/unwrapLPs')
+
+const fantomVault_01 = "0x9e4105f9e2284532474f69e65680e440f4c91cb8";
+const chain= 'fantom'
+
+const fantomTVL = async (timestamp, _block, { [chain]: block}) => {
```

Comment:
hi @ftm1337 you can use our helper for gmx tvl: 

    const { gmxExports } = require("../helper/gmx");

    module.exports = {
      fantom: {
        tvl: gmxExports({ vault: '0x9e4105f9e2284532474f69e65680e440f4c91cb8' }),
      },
    };

Reply:
oh thats nice! thanks fren!